.github/workflows/demos.yaml

```diff
@@ -1,36 +1,31 @@
 name: Demos
-
 on:
   workflow_dispatch:
-
+  pull_request:
+    types: [ labeled ]
 jobs:
   build-and-run-demos:
+    if: ${{ github.event.label.name == 'check-demos' }}
     name: Build and Run Demos
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v2
       with:
         go-version: v1.16
-
     - run: make build
-    
-    - uses: engineerd/setup-kind@v0.5.0
-      name: Install Kind
-      with:
-        skipClusterCreation: true
-        version: v0.10.0
 
-    - name: Prepare Demo tests
-      run:
-        sudo sysctl -w net.ipv6.conf.all.disable_ipv6=1 &&
-        sudo sysctl -w net.ipv6.conf.default.disable_ipv6=1 &&
-        sudo sysctl -w net.ipv6.conf.lo.disable_ipv6=1 &&
+    - name: Install latest version of Kind
+      run: |
+        GO111MODULE=on go get sigs.k8s.io/kind@v0.10.0
```

Comment:
Totally reasonable:

```
      - uses: engineerd/setup-kind@v0.5.0
        with:
          version: v0.10.0
```

Reply:
I don't have a strong opinion either way. Looking closer it looks like `setup-kind` also expects to create clusters by default, so it may not be a great fit for our use case.

I still think we should `go install` instead of `go get` since we know we'll be using go 1.16+.